docs/updating/migration-to-cdn/react.md

```diff
@@ -0,0 +1,455 @@
+```
+
+#### Step 2: Use `useCKEditorCloud` hook to load CKEditor&nbsp;5 from CDN
+
+`useCKEditorCloud` is a hook that allows you to load CKEditor&nbsp;5 from the CDN. It returns an object with the `CKEditor` and `CKEditorPremiumFeatures` properties. Here is an example of migrating the basic CKEditor&nbsp;5 React component:
+
+**Before:**
+
+```jsx
+import { CKEditor } from '@ckeditor/ckeditor5-react';
+import { ClassicEditor, Bold, Essentials, Italic, Mention, Paragraph, Undo } from 'ckeditor5';
+import { SlashCommand } from 'ckeditor5-premium-features';
+
+import 'ckeditor5/ckeditor5.css';
+import 'ckeditor5-premium-features/ckeditor5-premium-features.css';
+
+function App() {
+	return (
+		<CKEditor
+			editor={ ClassicEditor }
+			config={ {
+				toolbar: {
+					items: [ 'undo', 'redo', '|', 'bold', 'italic' ],
+				},
+				plugins: [
+					Bold, Essentials, Italic, Mention, Paragraph, SlashCommand, Undo
+				],
+				licenseKey: '<YOUR_LICENSE_KEY>', // Or 'GPL'.
+				mention: {
+					// Mention configuration
+				},
+				initialData: '<p>Hello from CKEditor 5 in React!</p>',
+			} }
+		/>
+	);
+}
+
+export default App;
+```
+
+**After:**
+
+```jsx
+import { CKEditor, useCKEditorCloud } from '@ckeditor/ckeditor5-react';
+
+function App() {
+	// Load CKEditor 5 from the CKEditor Cloud, using the `useCKEditorCloud` hook.
+	// It'll inject the CKEditor 5 scripts and styles into your document head and after
+	// successful loading, it'll return the `CKEditor` and `CKEditorPremiumFeatures` objects.
+	const cloud = useCKEditorCloud( {
+		version: '{@var ckeditor5-version}', // Required. The version of CKEditor 5 to load.
+		premium: true // Optional. Set to `true` if you want to use premium features.
+	} );
+
+	if ( cloud.status === 'loading' ) {
+		return <div>Loading...</div>;
+	}
+
+	if ( cloud.status === 'error' ) {
+		console.error( cloud.error );
+
+		return <div>Error!</div>;
+	}
+
+	// Pick the CKEditor 5 plugins you want to use.
+	const {
+		ClassicEditor, Bold, Essentials, Italic,
+		Mention, Paragraph, SlashCommand, Undo
+	} = cloud.CKEditor;
+
+	const { SlashCommand } = cloud.CKEditorPremiumFeatures;
+
+	return (
+		<CKEditor
+			editor={ ClassicEditor }
+			config={ {
+				toolbar: {
+					items: [ 'undo', 'redo', '|', 'bold', 'italic' ],
+				},
+				plugins: [
+					Bold,
+					Essentials,
+					Italic,
+					Mention,
+					Paragraph,
+					SlashCommand,
+					Undo
+				],
+				licenseKey: '<YOUR_LICENSE_KEY>', // Or 'GPL'.
+				initialData: '<p>Hello from CKEditor 5 in React!</p>',
+			} }
+		/>
+	);
+}
+```
+
+#### Step 3 (Optional): Migrate the CKEditor&nbsp;5 React testing suite
+
+If you have any tests that use CKEditor&nbsp;5 objects, you need to update them to use the  `loadCKEditorCloud` function. Here is an example of migrating a test that uses the `ClassicEditor` object:
+
+**Before:**
+
+```javascript
+import { ClassicEditor, ... } from 'ckeditor5';
+import { EasyImage, ... } from 'ckeditor5-premium-features';
```

Comment:
Let's not use Easy Image it wil be deprecated, if we choose to use premium plugins, align them across all migration guides.